docs/governance/minutes/2019-05-15.rst

```diff
@@ -0,0 +1,284 @@
+
+Many of you have heard me say that “I believe in open source the way I believe in democracy”. But like democracy, open source is evolving as new contributors, new technology, and new cultures start to participate more. Yellowbrick is representative of what open source can achieve and what it does both for and to the people who maintain it. As we move forward from here, the choices we make may have an impact beyond our project and perhaps even beyond Python and I hope that you, like me, find that both exciting and terrifying.
+
+Today we are here to discuss how we will conduct the project toward a version 1.0 release, a significant milestone, while also mentoring a Google Summer of Code Student and managing the many open pull requests and issues currently outstanding. The details matter, but they must also be considered within a context. Therefore I would like to personally commission you all toward two goals that set that context:
+
+1. **To be a shining example of what an open source project should be.**
+2. **To think strategically of how we can support a community of both users and maintainers.**
+
+I’m very much looking forward to the future, and am so excited to be doing it together with you all.
+
+PyCon 2019 Debrief
+~~~~~~~~~~~~~~~~~~
+
+Yellowbrick had an open source booth at PyCon 2019 for the second time, manned by Larry, Nathan, Prema, and Kristen. There were lots of visitors to the booth who were genuinely excited about using Yellowbrick. Next year it would be great to have a large banner and vertical display of visualizers to help people learn more. Guido von Rossum even had lunch at the booth, which sparked a lot of attention! Special thank you to Tony for his financial support including shirts, stickers, transportation, and supplies.
+
+We also hosted two days of development sprints with 13 sprinters and 9 PRs. The majority of sprinters were there for both days. The sprints included a wide variety of people including a high schooler and even a second year sprinter. Guido von Rossum also stopped by during the sprints. Prema announced the sprints on stage and there is a great picture of her and Kristen. Special thank you to Daniel for creating the cheatsheets, which cut down a lot of explaining, and to Nathan for the appetizers. Beets are delicious!
+
+Hot takes from PyCon:
+
+- Cheatsheets were a huge success and very important
```

Comment:
Change this to `- Cheatsheets were a huge success and very important.  It would be great to create a cheatsheet for "Contributing to Yellowbrick" ahead of next year's sprints.

Reply:
Added on the line below for space.